code/nodes/featureSignificance/funcs.q

```diff
@@ -0,0 +1,70 @@
+\d .automl
+
+// Definitions of the main callable functions used in the application of .automl.featureSignificance
+
+// @kind function
+// @category featureSignificance
+// @fileoverview Extract feature significant tests and apply to feature data
+// @param cfg   {dict} Configuration information assigned by the user and related to the current run
+// @param feats {tab}  The feature data as a table 
+// @param tgt   {(num[];sym[])} Numerical or symbol vector containing the target dataset
+// @return {sym[]} Significant features or error if function does not exist
+featureSignificance.applySigFunc:{[cfg;feats;tgt]
+  sigFunc:@[get;cfg`sigFeats;{'"Feature significance function ",x," not defined"}];
+  sigFunc[feats;tgt]
+  }
+
+// @kind function
+// @category featureSignificance
+// @fileoverview Apply feature significance function to data post feature extraction
+// @param cfg   {dict}          Configuration information assigned by the user and related to the current run
+// @param feats {tab}           The feature data as a table 
+// @param tgt   {(num[];sym[])} Numerical or symbol vector containing the target dataset
+// @return      {sym[]}         Significant features
+featureSignificance.significance:{[feats;tgt]
+  sigFeats:.ml.fresh.significantfeatures[feats;tgt;.ml.fresh.benjhoch .05];
+  if[0=count sigFeats;
+    sigFeats:.ml.fresh.significantfeatures[feats;tgt;.ml.fresh.percentile .25]];
+  sigFeats
+  }
+
+// @kind function
+// @category featureSignificance
+// @fileoverview Count how many significant columns were returned by significance tests
+// @param feats    {tab}   The feature data as a table 
+// @param sigFeats {sym[]} Significant columns
+// @return {sym[]} Significant columns
+featureSignificance.countCols:{[feats;sigFeats]
+  $[0<>count sigFeats;
+    sigFeats;
+    [-1"Feature significance extraction deemed none of the features to be important. Continuing with all features.";
```

Comment:
Not sure that we really need this one any more? Seems like it was a hangover from when we originally just used the benjamini hochberg test which could produce a case where no significant features were found. Now with the percentile stuff I don't think it's possible to make it run without determining something is significant?